.github/workflows/integration-test.yml

```
@@ -0,0 +1,40 @@
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --ignore-scripts
+
+      - name: Build
+        run: yarn compile && yarn run pack
+
+      - name: Run action
+        uses: ./
```

actionlint (1.7.12)

[error] invalid runner name "node12" at runs.using in "Setup Minikube" action defined at "/home/jailuser/git". valid runners are "composite", "docker", "node20", and "node24". see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs

(action)